### PR TITLE
fixes from testing basic example, focus language on DAGs

### PR DESCRIPTION
**Pull request by b5**
Screenshot 2024-07-03 at 5 20 40 PM

### CODE REVIEW

File: src/app/blog/iroh-dag-sync/page.mdx [hunk 1]
```diff
@@ -108,26 +90,29 @@ Or even more complex, only follow links in blocks where some sort of checksum or
 
 You could try to design a generic language to specify (deterministic) DAG traversals. A protocol that takes this approach is [graphsync](https://ipld.io/specs/transport/graphsync/). But especially for some of the more complex use cases mentioned above you would end up having to define a turing complete language.
 
-So I have decided to just implement a number of useful traversals of varying level of complexity, but allow implementing additional deterministic traversals in rust. Of course this means that sync will have to be tailored to the needs of the application, and different applications will have different ALPNs
-for their sync protocols
+So, we have decided to just implement a number of useful traversals of varying level of complexity, but allow implementing additional deterministic traversals in rust. Of course this means that sync will have to be tailored to the needs of the application, and different applications will have different ALPNs for their sync protocols.
+
+** **
```
Comment (rklaehn) [inline src/app/blog/iroh-dag-sync/page.mdx:95]: What's this for?

File: src/app/blog/iroh-dag-sync/page.mdx [hunk 2]
```diff
@@ -108,26 +90,29 @@ Or even more complex, only follow links in blocks where some sort of checksum or
 
 You could try to design a generic language to specify (deterministic) DAG traversals. A protocol that takes this approach is [graphsync](https://ipld.io/specs/transport/graphsync/). But especially for some of the more complex use cases mentioned above you would end up having to define a turing complete language.
 
-So I have decided to just implement a number of useful traversals of varying level of complexity, but allow implementing additional deterministic traversals in rust. Of course this means that sync will have to be tailored to the needs of the application, and different applications will have different ALPNs
-for their sync protocols
+So, we have decided to just implement a number of useful traversals of varying level of complexity, but allow implementing additional deterministic traversals in rust. Of course this means that sync will have to be tailored to the needs of the application, and different applications will have different ALPNs for their sync protocols.
```
Comment (rklaehn) [inline src/app/blog/iroh-dag-sync/page.mdx:93]: Not a big deal, but seems weird to use we when there was nobody but me involved.

File: src/app/blog/iroh-dag-sync/page.mdx [hunk 3]
```diff
@@ -108,26 +90,29 @@ Or even more complex, only follow links in blocks where some sort of checksum or
...
 The demo in [iroh-experiments](https://github.com/n0-computer/iroh-experiments/tree/main/iroh-DAG-sync)implements a simple unidirectional sync between two nodes. You can think of it as [sendme](https://crates.io/crates/sendme), but for DAGs.
 
 ## Generate some data
 
-We need a car file. You can just import some directory into ipfs and then export
-it as a car file. Make sure to use --raw-leaves to have a more interesting DAG.
+We need a car file. You can just import some directory into ipfs and then export it as a car file. Make sure to use `--raw-leaves` to have a more interesting DAG structure. You'll need [kubo](https://dist.ipfs.tech/#kubo) for this.
 
-Let's use the linux kernel sources as an example.
+Let's use the linux kernel sources as an example. Feel free to put any directory you like in here
```
Comment (rklaehn) [inline src/app/blog/iroh-dag-sync/page.mdx:105]: . missing

**Review by rklaehn — APPROVED** [review]

File: src/app/blog/iroh-dag-sync/page.mdx (hunk 1 above)
Comment (b5) [reply src/app/blog/iroh-dag-sync/page.mdx:95]: this creates a small horizontal rule in the final rendering to help distinguish the sections. On second consideration it should probably be removed. I'll take it out

File: src/app/blog/iroh-dag-sync/page.mdx (hunk 2 above)
Comment (b5) [reply src/app/blog/iroh-dag-sync/page.mdx:93]: It's in the name of consistent voicing. The post starts off with "we" and "you", and should not change from that point forward

**Pull request merged**